Resources/AddService/IPAddress.json

```diff
@@ -0,0 +1 @@
+{"Device": [{"IPAddress": "437XR1138R2"}, {"IPAddress": "437XR1138R3"}, {"IPAddress": "437XR1138R4"}, {"IPAddress": "183.25.16.12"}, {"url": "http://183.25.16.12:5000/redfish/v1/", "IPAddress": "183.25.16.12"}]}
```

Comment:
What is the purpose of this file?  These seem to be specific to your configuration.